src/logger.cr

```diff
@@ -0,0 +1,72 @@
+class Logger
+  property :level, :progname, :formatter
+
+  UNKNOWN = 5
+  FATAL = 4
+  ERROR = 3
+  WARN = 2
+  INFO = 1
+  DEBUG = 0
+
+  SEV_LABEL = %w(DEBUG INFO WARN ERROR FATAL ANY)
```

Comment:
Yes, I wanted to have a Logger::Level enum, but eventually ran into problems with the `SEV_LABEL`. Array keys are Int32 and, compiler will never consider the enum values them as Int32. I tried `Int32.cast(severity)` and a Hash like `{ Level::DEBUG => "DEBUG" }` but to no avail.

Sometimes, types are :boom: